skel/problem/answer_validators/answer_validator/answer_validator.cpp

```diff
@@ -0,0 +1,22 @@
+#include "validation.h"
+
+// This program will be called as
+// answer_validator input < ans
+//
+// You should verify the grammar of the answer file.
+// See input_validator.cpp for information on how to use the Validator class.
+// Furthermore you should check simple properties of the answer.
+
+// TODO: Remove these comments, and summarize your answer validator.
+
+int main(int argc, char *argv[]) {
+    // Set up the input and answer streams.
+    std::ifstream in(argv[1]);
+    // std::ifstream ans(argv[2]); // Only for custom checker.
```

Comment:
This line should probably be removed from the answer validator, since the answer file is passed via stdin, and not as an argument?